internal/clients/team6/iifo.go

```diff
@@ -1,134 +1,155 @@
 package team6
 
 import (
-	"math"
-
-	"github.com/SOMAS2020/SOMAS2020/internal/common/baseclient"
 	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+	"gonum.org/v1/gonum/stat/distuv"
 )
 
-func (c *client) MakeDisasterPrediction() shared.DisasterPredictionInfo {
-	// similar to the baseclient implement: mean prediction
-	pastDisasters := c.disastersHistory
-	teamsOfferingTo := []shared.ClientID{}
+func (c client) checkIsStochastic() bool {
```

Comment:
why is this not a ptr receiver?

Reply:
it's fine becasuse you are not mutating c but i was just wondering

Reply:
yeah just preventing me from changing some values accidently

Reply:
ah -- good that you know this :)